include/odp/api/spec/classification.h

```diff
@@ -530,11 +530,21 @@ typedef struct odp_cls_stats_capability_t {
  */
 typedef struct odp_cls_capability_t {
 	/** PMR terms supported by the classifier
-	 * A bit mask of one bit for each of odp_pmr_term_t
-	 */
+	 *
+	 * A bit mask of one bit for each of odp_pmr_term_t. */
 	odp_cls_pmr_terms_t supported_terms;
 
-	/** Maximum number of PMR terms */
+	/** Maximum number of single-term PMRs
+	 *
+	 * Depending on the implementation, using several/composite terms for a
+	 * single PMR may end up incurring more than one PMR element from this
+	 * total capacity. */
+	uint32_t max_pmr;
+
+	/** Maximum number of PMRs per CoS */
+	uint32_t max_pmr_per_cos;
+
+	/** Maximum number of terms per PMR */
 	uint32_t max_pmr_terms;
```

Comment:
As there were no validation tests for the field and as the previous documentation was ambigious, it's debatable whether the meaning has changed so ended up just "clarifying" the field. How this was interpreted differs between implementations, `linux-gen` was probably wrong. `max_terms_per_pmr` would probably be a better name, though will cause compilation errors if changed.

Reply:
Since we are doing a backward incompatible release anyway, it could make sense to use the better name. `available_pmr_terms` could also be removed altogether, since it's an odd leftover and doesn't seem to be used anywhere.

Reply:
I can change the name. If we are removing something, that should probably be done via the deprecation pipeline instead of just directly removing.

Reply:
We discussed with Petri about this and concluded that we could take a shortcut here and just remove the field, since it has not been really used anywhere.

Reply:
V3.